webcompat/templates/contact.html

```diff
@@ -2,12 +2,65 @@
 {% block title %}Contact{% endblock %}
 {% block body %}
 {% include "shared/nav.html" %}
-  <section class="grid">
+<section class="grid">
     <article class="grid-row">
-      <div class="grid-cell x3">
-        <h1 class="headline-1">TODO: Contact</h1>
-        <p>Et repellendus voluptatum nobis aut iure sint vel eaque. Nesciunt id architecto alias. Fugiat et quas explicabo. Aspernatur maiores explicabo aut. Autem autem quibusdam sit molestiae maiores blanditiis quis est. Ut laborum qui est dolorem itaque omnis sit incidunt.</p>
-      </div>
+		<div class="grid-cell x3">
+			<h1 class="headline-1 highlight">Say hello!</h1>
+		</div>
+		<div class="grid-cell x2">
+		    <p>
+		    	 No matter, if you want to <a href="./issues/new" title="Report a Bug">report a bug</a>, need help, <a href="./contributors" title="Want to Contribute">want to contribute</a>, let us know. Please keep our <a href="https://github.com/webcompat/webcompat.com/blob/master/CODE_OF_CONDUCT.md" title="Code of Conduct">Code of Conduct</a> in mind as we are an inclusive community. Please use gender-neutral terms (like "folks", "y'all" or "Wompats"), be respectful to each other which includes all of our channels. If you feel like there was a violation of the Code of Conduct, <a href="https://github.com/webcompat/webcompat.com/blob/master/CODE_OF_CONDUCT.md#enforcement" title="feel free to report a Code of Conduct violation">feel free to report</a> it anytime.
+		    </p>
+		    <p>We are a community and <span class="bold">we're here for you</span>.</p>
+		</div>
+		<div class="grid-cell x1">
+			<p>
+				Find our code <a href="https://github.com/webcompat" title="code on Github">on GitHub</a><br />
+		    	Find ways <a href="./contributors" title="Find ways to Contribute">to contribute</a><br /><br />
+		    	Find us <a href="https://twitter.com/webcompat/with_replies" title="Find us on Twitter">on Twitter</a> (all vendors)
+		    	Find us <a href="https://twitter.com/mozwebcompat/with_replies" title="Find us on Twitter">on Twitter</a> (just Mozilla related bugs)
+			</p>
+		</div>
+	</article>
+</section>
+<section class="grid">
+    <article class="grid-row">
+		<div class="grid-cell x1">
+			<h2 class="headline-2">Channels</h2>
+		    <ul>
+		    	<li>Join us 
+		    		<a href="https://wiki.mozilla.org/IRC#Connect_to_the_Mozilla_IRC_server" title="Join us on IRC">
+		    			on IRC
+		    		</a>
+		    	</li> 
+		    	<li>
+		    		Join us on our 
+		    		<a href="https://lists.mozilla.org/listinfo/compatibility" title="Join us on our mailing list">
+		    			mailing list
+		    		</a>
+		    	</li> 
+		    	<li>
+		    		Join our
+		    		<a href="https://webcompat.slack.com/" title="Join our Slack">
+		    			Slack
```

Comment:
😮

Reply:
not good?

Reply:
Heh no, we've chatted about this before. All good.